pubsub/redis/redis.go

```diff
@@ -367,21 +408,20 @@ func (r *redisStreams) handlePendingWithoutMessages(stream string, messageIDs ma
 			MinIdle:  r.metadata.processingTimeout,
 			Messages: []string{pendingID},
 		}).Result()
-		if err != nil && err != redis.Nil {
+		if err != nil && !errors.Is(err, redis.Nil) {
 			r.logger.Errorf("error claiming pending Redis message %s: %v", pendingID, err)
 
 			return
```

Comment:
should this be continue instead of return? That is try to claim other pending Messages instead of returning on the first one that is not a redis.Nil error ?

Reply:
Yes!  Good catch!